modules/caddyhttp/server.go

```diff
@@ -264,18 +263,6 @@ type Server struct {
 
 // ServeHTTP is the entry point for all HTTP requests.
 func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	// If there are listener wrappers that process tls connections but don't return a *tls.Conn, this field will be nil.
-	// TODO: Can be removed if https://github.com/golang/go/pull/56110 is ever merged.
```

Comment:
Whoa, 3 years later, it's finally in! 👏 Is it too early to drop this though, isn't it only in Go 1.25?

Reply:
That's why I leave this as a draft. I have yet to test this thoroughly.

Reply:
Yeah, this should be left in for now. Other parts seem fine though.